lib/module.js

```diff
@@ -1,19 +1,20 @@
 const { resolve } = require('path')
-const fs = require('fs-extra')
+const { copyFileSync, existsSync } = require('fs')
```

Comment:
I've used `fs-extra` to benefit from parallel file checking. We could theoretically use [`fs.promises`](https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fs_promises_api) for that but it won't make a huge difference I guess as we only check for two files 🤷🏻‍♂️

Reply:
Exactly, it is a simple check.